### PR TITLE
Improve wording of partial/unknown status

### PR DESCRIPTION
**Pull request by caugner**
See: #230

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -90,8 +90,8 @@
     "state": {
       "open": "Place open during lockdown",
       "open_adapted": "Place open during lockdown, with adapted hours",
-      "partial": "Place potentially open during lockdown (to be checked)",
-      "unknown": "Place with legal right to open during lockdown, but is it really open or closed ? Let us know",
+      "partial": "Place belongs to a brand whose places are partially open during lockdown. Is it open or closed? Let us know.",
+      "unknown": "Place may be allowed to open during lockdown. Is it open or closed? Let us know.",
       "closed": "Place closed during lockdown"
```
Comment (caugner) [inline locales/en.json:95]: By the way, do we really have to mention "lockdown" here?

* open = Place still open
* open_adapted = Place still open with adapted hours
* partial = Place belongs to a brand whose places are partially still open. Is it open or closed? Let us know.
* unknown = Place may be allowed to open. Is it open or closed? Let us know.
* closed = Place temporarily closed

Comment (datendelphin) [reply locales/en.json:95]: I think we don't need to mention lockdown, I find this version even better.

**Comment by PanierAvide** [comment]
@overflorian any opinion on English wording for these ?

**Comment by overflorian** [comment]
I think that it is important to mention "lockdown" to avoid any confusion.
Some people may understand "Place still open" as "open right now" (when they read the text) and not "open during lockdown". 
We specifically choose this wording in french to avoid any confusion.

File: locales/en.json (hunk 1 above)
Comment (caugner) [reply locales/en.json:95]: See [Florian's answer](https://github.com/osmontrouge/caresteouvert/pull/329#issuecomment-612933304) below.

**Comment by caugner** [comment]
Thanks, Florian, for the clarification. That makes sense, now I understand where this comes from!

In German we have in fact different words to describe whether shops are "still open" w.r.t. to their opening hours (i.e. "noch offen/geöffnet") or "still open" despite circumstances ("weiterhin offen/geöffnet"). That's why we don't need to say "trotz Ausgangsbeschränkung", because it is clear from the wording and the current context.

Let's discard my removal proposal then (I already resolved my conversation), but could we still improve the wording (see changes of this PR)?

Right now, `Place with legal right to open during lockdown, ...` is IMO misleading for e.g. restaurants. It's a strong statement and especially tricky when we don't follow up with (legal) changes right away.

**Comment by overflorian** [comment]
of course you can (and you should!) change the wording in German if it makes sense.

**Comment by caugner** [comment]
> of course you can (and you should!) change the wording in German if it makes sense.

@overflorian Let me rephrase my question: Are you comfortable with your app making legal statements? What if a business owner reads that and thinks they can open their restaurant? Wouldn't that get you into trouble?

In Germany, the "law" states that restaurants are not allowed to open, but that they are still allowed to operate delivery and takeaway services. That's not the same as having "the legal right to open".

And this English wording also affects Germany, because English users in Germany also get that label, except you allow pseudo locales per country like en-DE. 

What exactly is your concern with my proposed wording? (Note that it still mentions the lockdown.)

**Comment by overflorian** [comment]
that's indeed a tricky question. Let me investigate that and get back to you soon.

**Comment by overflorian** [comment]
OK I think you're right @caugner: our french wording is very specific to France.
@PanierAvide @francois2metz I'm not sure of the best way to solve this issue.

The proper way to do it would be to have a fr-FR wording for France : "lieu ayant l'autorisation d'être ouvert" and fr-world that would be the french wording that would be the base for worldwide translation. But it could also be that the base for translation becomes english?

**Comment by francois2metz** [comment]
> The proper way to do it would be to have a fr-FR wording for France : "lieu ayant l'autorisation d'être ouvert" and fr-world that would be the french wording that would be the base for worldwide translation. But it could also be that the base for translation becomes english?

All translations use the english wording, so we can differ it from french. There is a tricky part if a french living abroad (say Germany) is seeing the wording. It may be acceptable.

**Comment by overflorian** [comment]
I approve the changes of wording proposed by @caugner
